core/db_classes/EE_Form_Section.class.php

```diff
@@ -186,98 +206,130 @@ public function setBelongsTo(string $parent_UUID)
 
 
     /**
-     * HTML classes to be applied to this form section's container.
-     *
-     * @return string
+     * @return EE_Form_Element[]
      * @throws EE_Error
      * @throws ReflectionException
      */
-    public function htmlClass(): ?string
+    public function formElements(): array
     {
-        return $this->get('FSC_htmlClass');
+        return $this->form_elements ?: $this->getFormElements();
     }
 
 
     /**
-     * @param string $html_class
+     * @return EE_Form_Element[]
      * @throws EE_Error
      * @throws ReflectionException
      */
-    public function setHtmlClass(string $html_class)
+    private function getFormElements(): array
```

Comment:
Should this be `private`?

Reply:
i guess it could be public... although currently form sections are either related to another form section or  to an event, which are both represented by:

- `getChildFormSections()`
- `getFormSectionsForEvent()`

which set the appropriate where params for you, but doesn't hurt to do this